tests/tests_unit/test_utils/test_time.py

```diff
@@ -8,6 +8,7 @@
 from unittest import mock
 
 import pytest
+import pytz
```

Comment:
I thought you had to move this inside something marked with "dsl", but if it works it works 😅

Reply:
I have absolutely no idea... But it seems to test properly, so 🙄